src/Attachment.php

```diff
@@ -124,7 +124,13 @@ public function link()
      */
     public function path(): string
     {
-        return URLHelper::get_rel_path($this->src());
+        $src = $this->src();
+        // If src is a URL, convert it to a filesystem path first.
+        if (URLHelper::is_url($src)) {
```

Comment:
@Levdbas As you mentioned, this check is probably not needed, as `$src` should already be a URL.